lib/qx/tool/cli/ConfigSchemas.js

```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2019 The qooxdoo developers
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+     * Christian Boulanger (info@bibliograph.org, @cboulanger)
+
+************************************************************************ */
+
+/**
+ * This class contains constants used for locating and versioning
+ * of the different configuration files
+ */
+qx.Class.define("qx.tool.cli.ConfigSchemas", {
+
+  statics: {
+
+    library: {
+      filename: "Manifest.json",
+      schema: "https://qooxdoo.org/schema/Manifest.json/v2"
```

Comment:
the schema path looks a bit odd ... I would reather expect something like

https://qooxdoo.org/schema/Manifest-json-schema-v2.0.0.json

Reply:
no, it doesn't. it's just a placeholder. so whenever we/I get to actually create one, the URI will be replaced to point to where it placed (probably on GitHub).

Reply:
I created the ConfigSchema class to provide a central location for this kind of information, so that it doesn't clutter the code and creates a maintenance headache.